draft-api/src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -832,26 +832,35 @@ trait ConverterService {
         )
     }
 
-    private[service] def _stateTransitionsToApi(
+    private[service] def buildTransitionsMap(
         user: TokenUser,
         article: Option[Draft]
-    ): Map[String, Seq[String]] = {
-      StateTransitionRules.StateTransitions.groupBy(_.from).map { case (from, to) =>
-        from.toString -> to
-          .filter(_.hasRequiredRoles(user, article))
-          .map(_.to.toString)
-          .toSeq
-      }
-    }
+    ): Map[String, Seq[StateMachineStatus]] = DraftStatus.values
+      .filter(StateTransitionRules.StateTransitions.map(_.from).contains)
+      .map(from => {
+        val toStatuses = DraftStatus.values
+          .filter(StateTransitionRules.StateTransitions.map(_.to).contains)
+          .sorted
+          .map(to => {
+            val enabled = StateTransitionRules.StateTransitions
+              .find(transition => transition.from == from && transition.to == to)
+              .exists(_.hasRequiredRoles(user, article))
+
+            StateMachineStatus(name = to.toString, enabled = enabled)
+          })
+
+        from.toString -> toStatuses
```

Comment:
Dersom vi bestemmer oss for å legge `enabled: false` til slutt som vi snakket om, så kan vi gjøre denne endringen feks:
```suggestion
        val split = toStatuses.partition(_.enabled)
        from.toString -> (split._1 ++ split._2)
```